Source/WebCore/bindings/scripts/preprocess-idls.pl

```diff
@@ -217,6 +220,26 @@ END
 enum class DOMConstructorID : uint16_t {
 END
 
+my @structures = ();
+# Deprecated objects.
+push(@structures, "ObjCRuntimeMethod");
+push(@structures, "ObjCRuntimeObject");
+push(@structures, "ObjcFallbackObjectImp");
+push(@structures, "RuntimeArray");
+push(@structures, "RuntimeObject");
+push(@structures, "JSObservableArray");
+my $structuresHeaderCode = <<END;
+#include <wtf/FastMalloc.h>
+#include <wtf/Noncopyable.h>
+#include <JavaScriptCore/JSCInlines.h>
+
+#pragma once
+
+namespace WebCore {
+
+enum class DOMStructureID : uint16_t {
```

Comment:
It might be nice to have this enum in its own header file (DOMStructureID.h?) since a few of the places that include DOMStructures.h really only need the enum definition and don't need all the additional includes.